docs/diagrams/LogicClassDiagram.puml

```diff
@@ -8,7 +9,13 @@ package Logic {
 
 package Parser {
 Interface Parser <<Interface>>
-Class AddressBookParser
+Class "{abstract}\nWemeParser" as WemeParser
+Class MemeParser
+Class TemplateParser
+Class MemeCreationParser
+Class StatsParser
+Class ImportParser
+Class ExportParser
```

Comment:
Can we do with just `XYZParser` and add a note saying that it can be these? Like what is done for `XYZCommand`. Also `LogicManager` only seems to depend on `WemeParser` but not these concrete subclasses?